source/js/buyers-guide/homepage-c-slider.js

```diff
@@ -1,13 +1,28 @@
 import CREEPINESS_LABELS from "./components/creepiness-labels.js";
 
-// Height of a single frame, see the
-// ".current-creepiness" rule in homagepage.sccs
-const creepStep = 70;
+// Height of a single frame in the emoji sprite sheet,
+// see the ".current-creepiness" rule in homagepage.sccs
+const EMOJI_FRAME_HEIGHT = 70;
 
-// Total number of frames in our sprite sheet,
-// see the "./source/images/buyers-guide/faces/sprite.png" file
-const totalSteps = 40;
+// Total number of frames in our sprite sheet, see the
+// "./source/images/buyers-guide/faces/sprite.png" file.
+const SPRITE_FRAME_COUNT = 40;
 
+// Our threshold value for which average creepiness ratings
+// still count as "happy face" for the purpose of showing
+// the emoji while scrolling.
+//
+// Note: this is a cosmetic value for scroll only.
+const MINIMUM_HAPPINESS_RATING = 25;
+
+// Our threshold value beyond which everything is super
+// creepy by default.
+//
+// Note: this is a cosmetic value for scroll only.
+const MAXIMUM_CREEPINESS_RATING = 80;
```

Comment:
Are `MINIMUM_HAPPINESS_RATING` and `MAXIMUM_CREEPINESS_RATING` both fixed values unrelated to `EMOJI_FRAME_HEIGHT` and `SPRITE_FRAME_COUNT`? If there's any relationship between them I think it's worth adding some inline comments in case we decide to change the values for frame height and frame count in the future.

Reply:
There is zero relation between the two sets of numbers.